lib/Runtime/Base/JnDirectFields.h

```diff
@@ -554,183 +554,185 @@ ENTRY(__defineSetter__)
 ENTRY(__lookupGetter__)
 ENTRY(__lookupSetter__)
 ENTRY(__proto__)
-//Intl
-ENTRY(EngineInterface)
-ENTRY(raiseAssert)
-ENTRY(__minimumFractionDigits)
+
+// Intl
+#ifdef INTL_ENTRY
+#undef INTL_ENTRY
+#endif
+#define INTL_ENTRY(id, func) ENTRY(id)
+#include "../Library/IntlExtensionObjectBuiltIns.h"
```

Comment:
Does this mean the bytecode is different if Intl is enabled vs disabled?

Reply:
Oh, that's a good point. (Obviously applies to non-Intl bytecode only.) Might want to double check what happens if we generate bytecode for the other JS built-ins with/without Intl enabled. Likely the best thing to do here is to make sure these properties are created even if Intl is disabled. IIRC the purpose of splitting the file is to add a sort of namespace for Intl-related entries.

Reply:
I just checked the header file and it doesn't change depending on whether or not Intl is enabled. Might be good to add a comment to that header file explaining why it shouldnt be wrapped in an `#ifdef ENABLE_INTL_OBJECT`